app/controllers/pseuds_controller.rb

```diff
@@ -146,4 +146,14 @@ def destroy
 
     redirect_to(user_pseuds_url(@user))
   end
+
+  private
+
+  def pseud_params
+    params.require(:pseud).permit(
+      :user_id, :name, :description, :is_default, :icon, :delete_icon, :icon_file_name,
```

Comment:
But it is -- right here: https://github.com/otwcode/otwarchive/blob/master/app/views/pseuds/_pseuds_form.html.erb#L48